packages/gatsby-theme-docs/src/layouts/content-homepage.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useLayoutState from '../hooks/use-layout-state';
+import { useSiteData } from '../hooks/use-site-data';
+import { ContentPagination } from '../components';
+import LayoutApplication from './internals/layout-application';
+import LayoutHeader from './internals/layout-header';
+import LayoutSidebar from './internals/layout-sidebar';
+import LayoutMain from './internals/layout-main';
+import LayoutFooter from './internals/layout-footer';
+import LayoutPageWrapper from './internals/layout-page-wrapper';
+import LayoutPage from './internals/layout-page';
+import LayoutPageWithHero from './internals/layout-page-with-hero';
+import LayoutPageContent from './internals/layout-page-content';
+import PageContentInset from './internals/page-content-inset';
+
+const LayoutContentHomepage = (props) => {
+  const layoutState = useLayoutState();
+  const siteData = useSiteData();
+
+  return (
+    <LayoutApplication websitePrimaryColor={props.pageData.websitePrimaryColor}>
+      <LayoutSidebar
+        {...layoutState.sidebar}
+        {...layoutState.searchDialog}
+        siteTitle={siteData.siteMetadata.title}
+        isGlobalBeta={props.pageData.isGlobalBeta}
+        hasReleaseNotes={props.pageContext.hasReleaseNotes}
+      />
+      <LayoutMain
+        preventScroll={
+          layoutState.topMenu.isTopMenuOpen ||
+          layoutState.sidebar.isSidebarMenuOpen
+        }
+      >
+        <LayoutHeader
+          {...layoutState.searchDialog}
+          {...layoutState.topMenu}
+          siteTitle={siteData.siteMetadata.title}
+          excludeFromSearchIndex={props.pageData.excludeFromSearchIndex}
+        />
+        <LayoutPageWrapper id="top">
+          <LayoutPageWithHero
+            title={props.pageData.title}
+            heroBackgroundURL={props.heroBackground.publicURL}
+          >
+            <LayoutPage>
+              <LayoutPageContent>
```

Comment:
I think we need to override the `LayoutPageContent`, too  or find another way to give it `grid-area: page-content / page-content / page-navigation / page-navigation`  instead of `grid-area: page-content`.   

The design intent was to have the most relevant cards above the fold and rather drop the index nav column for that.

Reply:
I don't get your point. There is no index nav in this page, right?

Reply:
Yes, and the content column uses the width of the index nav for more side-by-side card or banner content in the design, but not yet in the implementation.

Reply:
Can you point me to the design that shows that?